lms/static/js/fixtures/dashboard/dashboard.html

```diff
@@ -48,10 +53,102 @@ <h3 class="course-title"><span>Introduction to Drinking Water Treatment</span></
                 <div class="wrapper-course-actions">
                     <div class="course-actions">
                         <div class="wrapper-action-more" data-course-key="course-v1:DelftX+CTB3365DWx+1T2016">
-                            <a href="#actions-dropdown-2" class="action action-more" id="actions-dropdown-link-2" aria-haspopup="true" aria-expanded="false" data-course-number="CTB3365DWx" data-course-name="Introduction to Drinking Water Treatment" data-dashboard-index="2">
-                                <span class="sr">Course options dropdown</span>
+                            <button type='button'class="action action-more" id="actions-dropdown-link-1" aria-haspopup="true" aria-expanded="false" data-course-number="CTB3365DWx" data-course-name="Introduction to Drinking Water Treatment" data-dashboard-index="1">
+                                <span class="sr">
+                                Course options for Introduction to Drinking Water Treatment
+                                </span>
                                 <i class="fa fa-cog" aria-hidden="true"></i>
+                            </button>
+                            <div class="actions-dropdown" id="actions-dropdown-1" aria-label="Additional Actions Menu">
```

Comment:
the aria-label here is forcing VO to treat this as a role=group.  since the aria-label on the ```<ul>``` below is also doing this, it requires a VO user to do a CMD+OPTION+SHIFT+DownArrow twice to actually get to the menu.  This is a odd implementation by VO, so its not really our fault here, but to minimize the difficulty, we should get rid of the aria-label here.  JAWS and NVDA do not do this, and they only read the aria-label on the ```<ul>``` below, so lets keep that one.